args_test.go

```diff
@@ -179,43 +303,67 @@ func TestArgsUsage(t *testing.T) {
 }
 
 func TestSingleOptionalArg(t *testing.T) {
-	cmd := buildMinimalTestCommand()
-	var s1 string
-	arg := &StringArg{
-		Min:         0,
-		Max:         1,
-		Destination: &s1,
-	}
-	cmd.Arguments = []Argument{
-		arg,
+	tests := []struct {
+		name     string
+		args     []string
+		argValue string
+		exp      []string
+	}{
+		{
+			name: "no args",
+			args: []string{"foo"},
+			exp:  []string{},
+		},
+		/*{
+			name: "no arg with def value",
+			args: []string{"foo"},
+			exp:  []string{"bar"},
+		},*/
```

Comment:
Should this case be handled? Currently the test fails when it is included

Reply:
Added